backend/Makefile

```diff
@@ -39,7 +42,7 @@ deploy:
 			-g ${RESOURCEGROUP} \
 			-n backend \
 			--query clientId -o tsv) && \
-	DB_URL=$$(az cosmosdb show -n ${DB_NAME} -g ${RESOURCEGROUP} --query documentEndpoint -o tsv) && \
+	DB_URL=$(az cosmosdb show -n ${DB_NAME} -g ${RESOURCEGROUP} --query documentEndpoint -o tsv) && \
```

Comment:
I'm always confused by single and double dollar. Was this broken before?

Reply:
It is a mistake, should be double dollar